index.html

```diff
@@ -10,39 +10,59 @@
 </head>
 
 <body>
-    <!-- Header start  -->
-    <!-- Header end -->
-
-    <!-- Hero start  -->
-    <!-- Hero end -->
-
-    <!-- Personal details start  -->
-    <!-- Personal details end -->
-
-    <!-- Offered services start  -->
-    <!-- Offered services end -->
-
-    <!-- Counter  start  -->
-    <!-- Counter end -->
-
     <!-- Latest featured projects start  -->
+    <div class="container">
+        <div class="row">
+            <h2>Our Latest Featured projects</h2>
+            <p>Who are in extremely love with eco friendly system.</p>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <ul>
+                <li class="active" data-filter="all">All</li>
+                <li>VECTOR</li>
+                <li>RASTER</li>
+                <li>UI/UX</li>
+                <li>PRINTING</li>
+            </ul>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-12 col-md-6 col-lg-4">
```

Comment:
```suggestion
            <div class="col-12 col-sm-4 col-lg-4">
            
```
per anksti sukrenta į stulpelį lyginant pagal originalą tai reiktų pakeist visus 6 iš md-6 į sm-4